DragonKitchen/DragonKitchen/Models/CurrentPageModel.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  CurrentPageModel.swift
+//  DragonKitchen
+//
+//  Created by 박상원 on 2023/05/11.
+//
+
+import Foundation
+
+enum PageName: Int {
+    case Eye = 1
+    case Hand = 2
+    case Ear = 3
+    case Nose = 4
+    case Mouth = 5
```

Comment:
enum에 들어가는 case의 경우 lowerCamelCase를 쓰는게 일반적인 것 같아요! 😊
```suggestion
    case eye = 1
    case hand = 2
    case ear = 3
    case nose = 4
    case mouth = 5
```